infrastructure/code/template/details.html

```diff
@@ -396,6 +660,12 @@ <h2 class="severity-title">{{.IssueTitle}}</h2>
       </div>
       <a class="lesson-link styled-link" target="_blank" rel="noopener noreferrer" href="{{.LessonUrl}}">Learn about
         this vulnerability</a>
+      <div class="learn--link is-external">
+        <svg width="9" height="9" viewBox="0 0 9 9" xmlns="http://www.w3.org/2000/svg" fill="none">
+          <path d="M4.99998 0L6.64648 1.6465L3.14648 5.1465L3.85348 5.8535L7.35348 2.3535L8.99998 4V0H4.99998Z" fill="#888"/>
```

Comment:
Adds the icon that marks the learn link as external

Reply:
What's your opinion about moving all the SVGs to `infrastructure/code/code_html.go` together with:
- Severity icons
- GitHub icon (I should have called it GitHub icon, sorry 🙏 )
- Lesson icon

https://github.com/snyk/snyk-ls/blob/4850808bc5da0b297fbc565b21f335999bd61cc3/infrastructure/code/code_html.go#L271-L286

Reply:
We need to wrap the SVG with the a tag so it becomes clickable:

Reply:
What is the reason for having the icons in there? I chose not to put it in that file because I didn't see the point of it.

Reply:
The icon was not clickable in the old view either, so I didn't make it clickable here

Reply:
- In the current implementation, the icon is clickable:

https://github.com/snyk/snyk-ls/assets/1948377/3c6a40fe-01cb-4f08-8406-53f9cbde82f0


- **SVG location**: with the SVGs and additional styling, the `details.html` has almost 1000 lines. Modularizing some bits makes the file more manageable and easier to maintain:

  - **Reusability**: Breaking down the code into smaller components makes it easier to understand, maintain, and update reducing the complexity of the `details.html` file.
  - **Scalability**: Moving SVGs to a separate file allows for reusability across the project, ensuring consistency and reducing redundancy. As and example of reusability, you wisely commented in [here](https://github.com/snyk/snyk-intellij-plugin/pull/535#discussion_r1621929683): "_in the future we will have more than Snyk Code styling but still want to share stylesheets or colours across Snyk OSS, Snyk IaC, and Snyk Container_".
  - **Collaboration**: Clean, modular code improves collaboration among team members by reducing merge conflicts and making it easier to work on the same project.

Reply:
Having the SVGs in the Go code is not the nicest: you don't benefit from the HTML syntax highlighting and we're only moving the complexity from one file to another, which is not really the right file to hold this kind of code. I had a look at other ways to do this. Maybe when we end up re-using this code for other product lines we could look at doing what they suggest here: https://stackoverflow.com/questions/24093923/optimising-html-template-composition. For now I'll do what you suggested to be consistent 👍